websites/D/DualFM/presence.ts

```diff
@@ -32,7 +32,7 @@ presence.on("UpdateData", async () => {
     smallImageKey: "dualfm-play-v2"
   };
 
-  const toggleelaspe = await presence.getSetting("toggleelapse");
+  const toggleelaspe = await presence.getSetting("toggleelapse") || true;
   const changedetails = await presence.getSetting("changedetails");
   const changestate = await presence.getSetting("changestate");
   const changesmalltext = await presence.getSetting("changesmalltext");
```

Comment:
give default value

Reply:
They already have a default value below, or is that irrelevant and I still need to define it in the variable itself?

Reply:
They will give the same error probs as what the first one did, so just do it to be sure.
Btw to fix the error you just have to change a setting (turn off and back on)

Reply:
update your fork too

Reply:
Updated the fork and that's what we found, we found that toggling the 'Toggle Elapse' setting seemed to fix it but it's a little inconvenient for some users haha. I've just added the some default values to the variables, hopefully all will be okay. Thank you for your assistance Bas and feel free to let me know if there's anything else that needs changing.